tools/test.py

```diff
@@ -31,7 +31,10 @@ def main(argv):
 
     enable_ansi_colors()
 
-    http_server.spawn()
+    try:
+        http_server.spawn()
+    except:
+        "Warning: another http_server instance is running"
```

Comment:
Should this be printing?

Though, the problem is that if you run `spawn` twice it doesn't raise (it just dumps the stacktrace). So I think handling this should be in the http_server module...

However, I am not convinced silently ignoring is the right thing to do, why is it required?

Reply:
My bad, missed a `print` during copy-paste.
I could actually move the other `spawn` into `main` of `benchmark.py` to avoid duplicate call to `spawn`, should be a better solution

Reply:
The pythonic thing to do here is use `with` but atm that is probably overkill.
Perhaps easier is to [throw exception in spawn](https://github.com/denoland/deno/blob/591174a686bc35c86b2100d660c7f066598a8829/tools/http_server.py#L28) if `not thread.is_alive()` ?

This is an aside to this PR :)

Reply:
see #833